Library/Formula/json-fortran.rb

```diff
@@ -15,9 +15,9 @@ class JsonFortran < Formula
 
   option "with-unicode-support", "Build json-fortran to support unicode text in json objects and files"
   option "without-test", "Skip running build-time tests (not recommended)"
-  option "without-robodoc", "Do not build and install ROBODoc generated documentation for json-fortran"
+  option "without-ford", "Do not build and install FORD generated documentation for json-fortran"
```

Comment:
OK, this seems to do the trick. FORD is listed as a build dependency but passing `--without-docs` does not trigger FORD to be downloaded and installed.